islandora.module

```diff
@@ -58,7 +58,9 @@ define('ISLANDORA_DATASTREAM_INGESTED_HOOK', 'islandora_datastream_ingested');
 define('ISLANDORA_DATASTREAM_MODIFIED_HOOK', 'islandora_datastream_modified');
 define('ISLANDORA_DATASTREAM_PURGED_HOOK', 'islandora_datastream_purged');
 define('ISLANDORA_INGEST_STEP_HOOK', 'islandora_ingest_steps');
+// ISLANDORA_DERVIATIVE_CREATION_HOOK is depricated. Should be removed after next release.
```

Comment:
A few issues:
- line is too long (break it across multiple lines)
- "deprecated", not "depricated"
- Should indicate what the current/next release will/should be to avoid making people gauge what the "next release" is based on when the commit was made... Since this is being pushed into the release, could say, "as of 7.x-1.5" or something of the like.